<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Logo">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>10</y>
      <width>811</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/logo/lg.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="CheckProgress">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>781</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::Direction::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QLabel" name="CheckAuthorizationLabel">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>530</y>
      <width>781</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Authorization...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LoginLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Login:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="PasswordLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="SignOutButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>540</y>
      <width>371</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SignInButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>371</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign in</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Login">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>7</y>
      <width>701</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Password">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>47</y>
      <width>701</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="LoginLabelReg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Login:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="PasswordLabelReg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="EmailHintLabelReg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>791</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(106, 100, 10);</string>
    </property>
    <property name="text">
     <string>It is recommended to specify a real email address so that in case of loss of access to the account, it can be restored</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="RegisterButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>528</y>
      <width>781</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EmailReg">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>701</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="EmailLabelReg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="LoginReg">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>701</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="PasswordReg">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>701</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
